### DISCUSSION TITLE
I can't pass the verification after 25 times

### DISCUSSION THREAD

**Discussion by LonelyAlan62** [community/community#52358] [GitHub Education]
I can't pass the checking. On the email I get "Your application cannot be reviewed until you fix the following:
You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio."
I have search for similar questions and solutions in the GitHub Community. But I can't find any useful answers to my question.

After improving bio and logging back into my github account, it encountered the following error:
“Unfortunately, we weren't able to approve your application.”
and “😢 @LonelyAlan62
Hey Zhang Tianrui, Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Apr 9, 2023 for @LonelyAlan62.Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved.Once you have resolved the issues with your sumbission, visit us again and reapply. Have questions? Join the GitHub Education Community for answers. 💖 The GitHub Education Team “
My files and steps are completely free from the errors mentioned in this website address, which I have repeatedly confirmed.
What should I do to pass?

**Comment by IBRAHIMtaiyeroheemat** [comment]
I am having same problem. Please help me.

**Comment by xqe2011** [comment]
same problem. I have tried 30 times. Either a screenshot from CHSI or an English transcript is acceptable.

**Reply by IBRAHIMtaiyeroheemat** [reply]
How will I get a transcript

**Reply by xqe2011** [reply]
I don't know, haha. Different schools have different ways. My school offers transcript in PDF format online. You can try contacting the Academic Affairs Office at your school.
